app/assets/stylesheets/shared/footer.scss

```diff
@@ -54,23 +54,28 @@
   width: 100%;
 
   input {
-    flex-grow:1;
+    border-top-left-radius: $form-border-radius;
```

Comment:
Properties should be ordered border-bottom-left-radius, border-bottom-right-radius, border-top-left-radius, border-top-right-radius, flex-grow, height, vertical-align